src/main/java/org/spongepowered/common/config/category/TileEntityActivationModCategory.java

```diff
@@ -33,15 +33,16 @@
 @ConfigSerializable
 public class TileEntityActivationModCategory extends ConfigCategory {
 
-    @Setting(value = "enabled", comment = "If 'false', tileentity activation rules for this mod will be ignored and always tick.")
+    @Setting(value = "enabled",
+            comment = "If 'false', tileentity activation rules for this mod will be ignored and always tick.")
     private boolean isEnabled = true;
-    @Setting(value = "default-block-range")
+    @Setting(value = "default-block-range", comment = "???")
     private Integer defaultBlockRange;
-    @Setting(value = "default-tick-rate")
+    @Setting(value = "default-tick-rate", comment = "???")
```

Comment:
same as previous `Default tick rate used for all tileentities unless overridden.`